libs/wire-api/src/Wire/API/Routes/Public/Spar.hs

```diff
@@ -124,6 +125,7 @@ type APIINTERNAL =
   "status" :> Get '[JSON] NoContent
     :<|> "teams" :> Capture "team" TeamId :> DeleteNoContent
     :<|> "sso" :> "settings" :> ReqBody '[JSON] SsoSettings :> Put '[JSON] NoContent
+    :<|> "scim" :> "userinfos" :> ReqBody '[JSON] UserSet :> Post '[JSON] ScimUserInfos
```

Comment:
Unrelated to this PR, but why is `APIINTERNAL` all-capsed (probably a question i should ask myself), and why is it defined in `.../API/Public.hs`?  Oh well.